nav2_bringup/README.md

```diff
@@ -0,0 +1,65 @@
+# Bringup
+
+## Before starting:
```

Comment:
I'm not clear if this is a README for the bring-up of the Navigation 2 system or just describing how to bring up gazebo. The location in nav2_bringup would indicate that it should describe how to bring up the whole system. We should also include instructions for starting the nav2 system.